docs: restructure and apply diataxis to OCI docs (#213)
canonical/ubuntu-cloud-docs: oci/oci-how-to/getting-started.rst

> Getting Started

Reviewer: ```suggestion
Getting started
```

***************

This is where to find the Ubuntu container images:

.. tabs::

	.. tab:: Docker Hub

         .. code-block:: bash

            docker pull ubuntu:latest
            

	.. tab:: Amazon ECR

         .. code-block:: bash
            
            docker pull public.ecr.aws/ubuntu/ubuntu:latest

	.. tab:: Microsoft ACR

         .. code-block:: bash
            
            docker pull ubuntu.azurecr.io/ubuntu:latest

	.. tab:: IronBank

         .. note::

>             Please note that these Ubuntu container images may have special terms of use and fall under Ubuntu Pro. Please reach out if you would like to use the Ubuntu container images from IronBank.

Reviewer: Who/where should users reach out to in this case? Could we add a link?

Author: We already have some links in the footer and the Project into at the index page.

I'm adding an email link nonetheless

            

Refresh rate
------------

> The Ubuntu base container images can be automatically updated multiple times a

Reviewer: What do we mean here by Ubuntu base container images? Are these basic/standard container images, or are we talking about Ubuntu Base?

Author: It's not the Ubuntu Base. Since containers are typically overlayed on top of each other in order to create other container images, sometimes we refer to this as the "Ubuntu base container image", to emphasize on the fact that is it different from other container images like `ubuntu/kafka` (which is an Ubuntu**-based** container image)

Author: I'm open to change this though if you have any ideas

Reviewer: I think that since you removed that one reference to Ubuntu Base, I think the Ubuntu base image terminology is less problematic. My main concern was having two things of the same name being potentially ambiguous and confusing, but I think that's resolved now. This is now fine to stay as it is.

day, depending on whether there have been upstream changes in the Ubuntu archives
affecting the packages that constitute the container image.

**Exception**: the only exception is the official Ubuntu image in Docker Hub which,

> due to the Docker's policies, cannot be submitted too regularly, and thus are

Reviewer: ```suggestion
due to Docker's policies, cannot be submitted too regularly, and so are
```
